doc/misc/npm-scripts.md

```diff
@@ -3,57 +3,90 @@ npm-scripts(7) -- How npm handles the "scripts" field
+* `prepublishOnly`:
+  * Runs BEFORE the package is prepared and packed, ONLY on `npm publish`. (See
   below.)
-* prepack:
-  run BEFORE a tarball is packed (on `npm pack`, `npm publish`, and when
+* `prepack`:
+  * Runs BEFORE a tarball is packed (on `npm pack`, `npm publish`, and when
   installing git dependencies)
-* postpack:
-  Run AFTER the tarball has been generated and moved to its final destination.
-* publish, postpublish:
-  Run AFTER the package is published.
-* preinstall:
-  Run BEFORE the package is installed
-* install, postinstall:
-  Run AFTER the package is installed.
-* preuninstall, uninstall:
-  Run BEFORE the package is uninstalled.
-* postuninstall:
-  Run AFTER the package is uninstalled.
-* preversion:
-  Run BEFORE bumping the package version.
-* version:
-  Run AFTER bumping the package version, but BEFORE commit.
-* postversion:
-  Run AFTER bumping the package version, and AFTER commit.
-* pretest, test, posttest:
-  Run by the `npm test` command.
-* prestop, stop, poststop:
-  Run by the `npm stop` command.
-* prestart, start, poststart:
-  Run by the `npm start` command.
-* prerestart, restart, postrestart:
-  Run by the `npm restart` command. Note: `npm restart` will run the
+* `postpack`:
+  * Runs AFTER the tarball has been generated and moved to its final destination.
+* `publish`, `postpublish`:
+  * Runs AFTER the package is published.
+* `preinstall`:
+  * Runs BEFORE the package is installed
+* `install`, `postinstall`:
+  * Runs AFTER the package is installed.
+* `preuninstall`, `uninstall`:
+  * Runs BEFORE the package is uninstalled.
+* `postuninstall`:
+  * Runs AFTER the package is uninstalled.
+* `preversion`:
+  * Runs BEFORE bumping the package version.
+* `version`:
+  * Runs AFTER bumping the package version, but BEFORE commit.
+* `postversion`:
+  * Runs AFTER bumping the package version, and AFTER commit.
+* `pretest`, `test`, `posttest`:
+  * Run by the `npm test` command.
+* `prestop`, `stop`, `poststop`:
+  * Run by the `npm stop` command.
+* `prestart`, `start`, `poststart`:
+  * Run by the `npm start` command.
+* `prerestart`, `restart`, `postrestart`:
+  * Run by the `npm restart` command. Note: `npm restart` will run the
   stop and start scripts if no `restart` script is provided.
-* preshrinkwrap, shrinkwrap, postshrinkwrap:
-  Run by the `npm shrinkwrap` command.
+* `preshrinkwrap`, `shrinkwrap`, `postshrinkwrap`:
+  * Run by the `npm shrinkwrap` command.
 
-Additionally, arbitrary scripts can be executed by running `npm
-run-script <stage>`. *Pre* and *post* commands with matching
-names will be run for those as well (e.g. `premyscript`, `myscript`,
-`postmyscript`). Scripts from dependencies can be run with `npm explore
-<pkg> -- npm run <stage>`.
+### Life Cycle
+
+npm commands such as `npm install`, `npm publish` will fire life cycle hooks as specified in your `package.json` file. These hooks are as follows for each command.
```

Comment:
I'd change the comma to "and": "npm commands such as `npm install` and `npm publish`..."